ada_2_fhir-r4/zibs2020/ada_instance/nl-core-TreatmentObjective-01.xml

```diff
@@ -0,0 +1,37 @@
+    <behandeldoel conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.13.5.1" id="bd-1">
+        <onderwerp conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.1">
+            <patient-id value="111222333" root="2.16.840.1.113883.2.4.6.3" conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.2"/>
+        </onderwerp>
+        <referenties conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.3">
+            <functionele_of_mentale_status conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.1" id="fom-1">
+                <referenties conceptId="2.16.840.1.113883.2.4.3.11.999.121.1.3">
+                    <medisch_hulpmiddel conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.1" id="mh-1">
+                        <product conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.2">
+                            <product_type code="462777007" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED Clinical Terms" displayName="elektrisch scheerapparaat" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.3"/>
+                        </product>
+                        <product_omschrijving value="Ergonomisch elektrisch scheerapparaat" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.13"/>
+                        <begin_datum value="2015-02-23" datatype="datetime" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.10.1.11"/>
+                    </medisch_hulpmiddel>
+                </referenties>
+                <status_naam code="284888003" codeSystem="2.16.840.1.113883.6.96" displayName="kan zich scheren" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.2"/>
+                <status_waarde code="284888003" codeSystem="2.16.840.1.113883.6.96" displayName="in staat om zich te scheren" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.3"/>
+                <status_datum value="2015-02-23" datatype="datetime" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.6"/>
+                <toelichting value="Zou mooi zijn zo snel mogelijk gebruik te kunnen maken van het nieuwe scheerapparaat." conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.4"/>
+                <hulpmiddel conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.4.26.5">
+                    <medisch_hulpmiddel value="mh-1" datatype="reference" conceptId="2.16.840.1.113883.2.4.3.11.60.121.2.142"/>
+                </hulpmiddel>
+            </functionele_of_mentale_status>
+            <probleem conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.5.1.1" id="pb-1">
+                <probleem_type conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.5.1.8" value="6" code="116223007" codeSystem="2.16.840.1.113883.6.96" displayName="Complicatie"/>
+                <probleem_naam code="86044005" codeSystem="2.16.840.1.113883.6.96" displayName="amyotrofische laterale sclerose" conceptId="2.16.840.1.113883.2.4.3.11.60.40.1.5.1.3"/>

```

Comment:
Use ICD-10 NL code G12.2 instead, since this SNOMED code is not part of ProbleemNaamCodelijst (see validation error).

Reply:
Changed for a SNOMED code from: 'SNOMED CT: ^11721000146100|Dutch nursing problem simple reference set| -- which should be included in the ProbleemNaamCodelijst.

Reply:
Still gives a validation error -- I think this has to do with the validator... ?

Reply:
The code 86044005 is not part of https://terminologie.nictiz.nl/terminology/snomed/viewRefset?id=11721000146100, so that seems to be the problem.

Reply:
@LuudSlagter  but I changed them: https://github.com/Nictiz/HL7-mappings/pull/83/commits/74e0bc6c86a5554a7916cd3888bc899d9c44f851

Reply:
Ah sorry, did not look at the right place! If you don't get these validation errors locally, then you can probably just ignore them (as locally the validation also includes the Dutch terminology and 11721000146100 is only part of the Dutch version of SNOMED).

Reply:
Mm... but I do get these validation errors locally. Could you give it a spin? :D